File: src/main/scala/com/nabijaczleweli/minecrasmer/compat/waila/WailaCompatRegisterer.java
package com.nabijaczleweli.minecrasmer.compat.waila;

import com.nabijaczleweli.minecrasmer.block.ComputerGeneric;
import mcp.mobius.waila.api.IWailaRegistrar;

public class WailaCompatRegisterer {
	public static final String pathToRegisterMethod = WailaCompatRegisterer.class.getName() + ".registerWailaCompats";

	@SuppressWarnings("unused")
	public static void registerWailaCompats(IWailaRegistrar registrar) {
		registrar.registerBodyProvider(ProviderComputer$.MODULE$, ComputerGeneric.class);
		registrar.registerStackProvider(ProviderComputer$.MODULE$, ComputerGeneric.class);
	}
}

File: src/main/scala/com/nabijaczleweli/minecrasmer/compat/waila/WailaCompatRegisterer.scala
package com.nabijaczleweli.minecrasmer.compat.waila

import com.nabijaczleweli.minecrasmer.util.StringUtils._
import net.minecraftforge.fml.relauncher.{Side, SideOnly}

@SideOnly(Side.CLIENT)
object WailaCompatRegisterer {
	final val pathToRegisterMethod = (getClass.getName before "$") + ".registerWailaCompats"

/*@SuppressWarnings(Array("unused"))
	def registerWailaCompats(registrar: IWailaRegistrar) {
		registrar.registerBodyProvider(ProviderComputer, classOf[ComputerGeneric])
		registrar.registerStackProvider(ProviderComputer, classOf[ComputerGeneric])

		registrar.registerBodyProvider(ProviderOverclocker, BlockAccessoryOverclocker.getClass)
		registrar.registerBodyProvider(ProviderAdditionalCPU, BlockAccessoryAdditionalCPU.getClass)
	}*/
}
